### DISCUSSION TITLE
keyball44-ランド剥離二カ所相談

### DISCUSSION THREAD

**Discussion by sarusaru-sa** [Yowkees/keyball#725] [Q&A]
keyball44のはんだ作業で2カ所ランド剥離した場合の回避方法を教えて頂けますでしょうか。
自作キーボード初心者です。
以下の青いカ所の相談です。
他の方のも参考に確認してみたのですが、イマイチ確信が持てず投稿させて頂きました。
大変お手数ですが、以下の青いカ所の回避方法を教えて頂けますでしょうか。
![IMG_1105](https://github.com/user-attachments/assets/1a1ca30a-94ea-4f87-b9f7-e184705dad36)
![IMG_1106](https://github.com/user-attachments/assets/d5a1cf27-7716-415a-93ed-9ab033fe0902)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

下記画像の同色どうしをエナメル線やゴム被覆導線等でショートしていただくと、正常にジャンパーできたのと同じ回路になり、復活できると思います。
右手側は少々距離が遠く申し訳ありません。

![KB44L_Jumpfix](https://github.com/user-attachments/assets/1dbdb8a4-fcea-43aa-ace4-ce9682d61def)
![KB44R_Jumpfix](https://github.com/user-attachments/assets/9d605240-c90a-4545-8ab8-3efd00b6b157)

以上、お手数ですがご対応お願いいたします。

**Reply by sarusaru-sa** [reply]
ありがとうございます。
早速試させていただきます!
取り急ぎ御礼まで。
